app/models/translation.py

```diff
@@ -0,0 +1,33 @@
+from datetime import datetime
+
+from app import db
+
+from app.models.chunk import Chunk
+
+
+class Translation(db.Model):
+    """
+    Translation is whole text requested by user for translation.
+    """
+    __tablename__ = 'translation'
+
+    translation_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(256))
+    created_ts = db.Column(db.DateTime)
```

Comment:
You can add default value here (`db.Column(db.DateTime, default=datetime.utcnow)`), and remove redundant `__init__` method.